### DISCUSSION TITLE
[ERROR] Only annotation mapping is supported by make:entity

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#42525] [Q&A] [answered]
I change doctrine.yaml orm type:
from:

```yaml
orm:
    mappings:
        App:
            type: annotation
```
			
to:

```yaml
orm:
    mappings:
        App:
            type: attribute
```
			
But show this error:

```
 [ERROR] Only annotation mapping is supported by make:entity, but the <info>App\Entity\Test</info> class uses a
         different format. If you would like this command to generate the properties & getter/setter methods, add your
         mapping configuration, and then re-run this command with the <info>--regenerate</info> flag.
```
		
How can I fix this problem?

**Comment by michaoj** [comment]
Seems like this post could help you https://github.com/vklux/maker-bundle-force-annotation

**Comment by javiereguiluz** [comment, marked as answer]
@karaka200 I'm afraid this problem cannot be fixed. We need to wait until MakerBundle merges the support of PHP attributes for entities. Check out this Pull Request to see the progress of this task: https://github.com/symfony/maker-bundle/pull/920

**Reply by karaka200** [reply]
Is there any problem if I use annotation in PHP 8?

**Reply by faizanakram99** [reply]
No, there is no problem.

Also you can convert annotations to attributes via rector

Search rectorphp/rector
